### PR TITLE
Add task to export all prizes

### PR DESCRIPTION
**Pull request by pferreira101**
This PR adds some functionalities to the existing export redeemable prizes task, including the list of prizes won in the Roulette. 
Also adds the possibility to generate a csv file.

### CODE REVIEW

File: lib/safira/store/store.ex [hunk 1]
```diff
@@ -89,7 +89,7 @@ defmodule Safira.Store do
     |> Map.fetch!(:redeemables)
     |> Enum.map(fn redeemable ->
       b = get_keys_buy(attendee.id, redeemable.id)
-      Map.put(redeemable, :quantity, b.quantity)
+      Map.put(%{name: redeemable.name}, :quantity, b.quantity)
```
Comment (joaop21) [inline lib/safira/store/store.ex:92]: This function `get_attendee_redeemables/1`, is used in AttendeeController, and its result is passed as an argument to the AttendeeControllerView, and then to RedeemableControllerView. So I don't think you should mess with its functionality. My suggestion would be to create a function for the desired purpose, or a mini function that uses this one and adds the step of changing the Map.

**Comment by jpcorreia99** [comment]
Since this task exports both the buys and the prizes from the roulette, maybe it should reflect that in the name of the task, since "buys" is already the name of a table and calling it only "export_buys" makes it look like it's only the content of that table that is exported. `export_buys_and_prizes` should do the trick.

File: lib/safira/store/store.ex (hunk 1 above)
Comment (pferreira101) [reply lib/safira/store/store.ex:92]: I'll simply undo this changes. The task should work anyway

**Comment by diogogmatos** [comment]
[Add task to export all prizes](https://trello.com/c/AqK5YU81/28-add-task-to-export-all-prizes)

**Comment by diogogmatos** [comment]
[Add task to export all prizes](https://trello.com/c/6Y4Aixtg/13-add-task-to-export-all-prizes)

**Pull request closed without merging**